javascript-data-model/order-history.js

```diff
@@ -0,0 +1,70 @@
+var orderHistory = [
+  {
+    orderNumber: '114-3941689-8772232',
+    shipTo: 'JS Masher',
+    billTo: 'JS Masher',
+    orderTotal: '$34.00',
+    paymentMethod: 'Visa 9090',
+    dateOrderPlaced: 'August 4, 2020',
+    dateOrderDelivered: 'Aug 8, 2020',
+    returnsClosed: 'Sep 7, 2020',
+    itemNumber: '11111',
```

Comment:
The only feedback I have for this is to create an `items` property which contains an array of objects which represent each individual item: 
```javascript
var orderHis = [
  {
    orderedDate: '2020-08-04',
    totalPrice: 34,
    shipTo: 'JS Masher',
    orderNum: '114-3941689-8772232',
    deliveredDate: '2020-08-08',
    orderedDate: '2-15-21',
    items:[{
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "electronics",
      title: "Blu Ray Player"
    },
    {
      type: "book",
      title: "How to forget"
    },
    {
      type: "book",
      title: "how to remember"
    }
  ]
  }
]
```

Reply:
Updated!